bin/install-site.sh

```diff
@@ -307,6 +323,15 @@ Pin-Priority: 50
 EOF
     fi
 
+    if [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
```

Comment:
I don't think we have any packages on debian.mysociety.org that depend upon specific ruby packages (or any that depend on ruby at all, from a quick look).